scripts/cv_api_master.py

```diff
@@ -388,5 +431,5 @@
     "path": "keras_cv/",
     "title": "KerasCV",
     "toc": True,
-    "children": [LAYERS_MASTER, MODELS_MASTER, BOUNDING_BOX_MASTER],
+    "children": [LAYERS_MASTER, MODELS_MASTER, BOUNDING_BOX_MASTER, LOSSES_MASTER],
```

Comment:
Do we also need metrics?

Reply:
We don't have any public class or any metrics exported in the Keras CV repo.